src/components/profile/profilemenu/desktop/ProfileMenu.js

```diff
@@ -217,22 +194,27 @@ export default function ProfileMenu() {
   if (!menus || !menus.loansAndReservations) return <></>;
 
   return (
-    <nav
-      className={styles.menu}
-      aria-label={`${Translate({
-        context: "profile",
-        label: "profileNavigation",
-      })}`}
-    >
-      <ul className={styles.menu}>
-        <MenuGroup
-          menus={menus}
-          name={menuItems[0]}
-          href="/profil/laan-og-reserveringer"
-        />
-        {/* more MenuLinks are coming soon */}
-        <MenuLink label={menuItems[1]} href="/profil/mine-biblioteker" />
-      </ul>
-    </nav>
+    <>
+      <nav
+        className={styles.menu}
+        aria-label={`${Translate({
+          context: CONTEXT,
+          label: "profileNavigation",
+        })}`}
+      >
+        <ul className={styles.menu}>
+          <MenuGroup
+            menus={menus}
+            name={menuItems[0]}
+            href={`/profil/${getTranslatedUrl(CONTEXT, menuItems[0])}`}
```

Comment:
Det var nu mere hele den relative url som jeg foreslog at lave en hjælpefunktion til, f.eks. `/profil/laan-og-reserveringer` som jeg tænkte man kunne lave en hjælpefunktion til. Så når/hvis UX'erne finder på, at vi skal ændre URL'en til f.eks. `brugerprofil/laan-og-reserveringer` i stedet for, så har vi dem ikke spredt rundt alle mulige steder. 

`getTranslatedUrl()` som funktion er fin nok (dog er navnet på variablen misvisende, for den foretager translate og encoding, men ikke noget med url som sådan). 

Specifikt var det jeg tænkte, noget i stil med `getProfileUrl(subpath)` som kan få f.eks. hele oversættelsen som du har lavet med `getTranslatedUrl()`, altså f.eks. `getProfileUrl(getTranslatedUrl(CONTEXT, menuItems[1]))`

Reply:
jeg har tilføjet en metode 👍